### ISSUE TITLE
Bug: `no-reversed-media-syntax` produces a destructive autofix inside inline `<code>` elements

### ISSUE THREAD

**Issue by minseonkkim** [eslint/markdown#715]
### Environment

ESLint version: ^10.8.1
@eslint/markdown version: v8.0.3
Node version: v22.15.0
npm version: v11.12.1
Operating System: Windows 11

### Which language are you using?

gfm

### What did you do?

<details>
<summary>Configuration</summary>

```
import markdown from "@eslint/markdown";

export default [
	{
		files: ["**/*.md"],
		plugins: { markdown },
		language: "markdown/gfm",
		rules: {
			"markdown/no-reversed-media-syntax": "error",
		},
	},
];
```
</details>

```md
<code>Object.create(<var>proxy</var>)[foo]</code>
```

I wrote inline code using an HTML `<code>` element so I could nest a `<var>` element for placeholder styling, which a plain backtick code span cannot express. Then I ran `npx eslint test.md` and `npx eslint test.md --fix`.

### What did you expect to happen?

No report, and no change from `--fix`.

The `(...)` and `[...]` sequences here are part of a code sample, not reversed link syntax, and the parenthesized portion consists entirely of raw HTML, which can never be a link destination.

The equivalent backtick code span is not reported:

```md
`Object.create(proxy)[foo]`
```

### What actually happened?

`npx eslint test.md`:

```
test.md
  1:20  error  Unexpected reversed syntax found. Use [label](URL) syntax instead  markdown/no-reversed-media-syntax

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

`npx eslint test.md --fix` rewrites the file destructively:

```diff
-<code>Object.create(<var>proxy</var>)[foo]</code>
+<code>Object.create[<var>proxy</var>](foo)</code>
```

The code sample is corrupted, and the fix creates a real Markdown link where the source had none.

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/stackblitz-starters-3tae5xpg?file=control.md

### Participation

- [x] I am willing to submit a pull request for this issue.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

### Analysis

`no-reversed-media-syntax` masks the ranges of `html`, `image`, `imageReference`, `inlineCode`, `linkReference`, and `inlineMath` nodes with spaces before running `reversedPattern` (the masking added in #597).

For this input, mdast produces:

| node type | value |
| --- | --- |
| `html` | `<code>` |
| `text` | `Object.create(` |
| `html` | `<var>` |
| `text` | `proxy` |
| `html` | `</var>` |
| `text` | `)[foo]` |
| `html` | `</code>` |

so the masked buffer becomes:

```
      Object.create(                )[foo]
```

The masking prevents characters inside the HTML from acting as delimiters, but the resulting run of spaces still matches the `label` capture group (`(?:\\.|[^()\\]|\([\s\S]*\))*`), so `(                )[foo]` is matched and reported.

The fixer then slices from `sourceCode.text` rather than from the masked buffer, so the original HTML is re-emitted inside a newly created link label. That is where the corruption comes from.

### Possible directions

1. Mask `html` nodes with a sentinel distinct from the other inline node types, and skip the match when the `label` or `url` range overlaps an `html` node. Trade-off: genuine reversed links whose label contains inline HTML, e.g. `(<b>bold</b>)[baz]`, would no longer be reported.
2. Keep reporting, but omit the fix when the match overlaps an `html` node. This removes the data loss without introducing false negatives.

Simply skipping whitespace-only labels would not work, because it would also suppress the intended behavior for masked `inlineCode`:

```md
Call (`fn`)[bar] here.
```

which is currently fixed to ``[`fn`](bar)`` as expected.

### Possible test cases

New `valid` case:

```js
"<code>Object.create(<var>proxy</var>)[foo]</code>",
```

Regression guard for the existing behavior:

```js
{
	code: "Call (`fn`)[bar] here.",
	output: "Call [`fn`](bar) here.",
	errors: 1,
},
```

### Related

- #483 — the lack of HTML element context is the underlying reason the rule cannot tell that it is inside a `<code>` element.
- #597 — the PR that introduced the masking this issue builds on.
- DavidAnson/markdownlint#2178 — the same asymmetry in markdownlint. The maintainer's position there is that CommonMark parses Markdown inside inline HTML, so reporting is defensible in general. That reasoning does not cover the autofix: the source contained no link, and the fix creates one.

Real-world context: this pattern comes from `mdn/translated-content`, whose style guide uses `<code>...<var>...</var>...</code>` for placeholder styling.

Disclosure: I'm a participant of [open source contribution program OSSCA](https://github.com/eslint-ossca)

**Comment by lumirlumir** [comment]
> Disclosure: I’m a participant in the [Open Source Contribution Program OSSCA](https://github.com/eslint-ossca): confirmed.

Thanks for the issue, but as noted in https://github.com/DavidAnson/markdownlint/issues/2178#issuecomment-5030774601, I don’t think this is a bug.

The `no-reversed-media-syntax` rule assumes that reversed link syntax is incorrect, based on the premise that the user’s original intention was to write valid link syntax, and applies this assumption as a heuristic.

For example, the following Markdown document is recognized as valid `Link` syntax according to the [playground](https://explorer.eslint.org/#eslint-explorer=v2.eyJzdGF0ZSI6eyJ0b29sIjoiYXN0IiwiY29kZSI6eyJqYXZhc2NyaXB0IjoiY29uc3QgaGkgPSA0MjtcblxuZGVidWdnZXI7XG5cbmRlYnVnZ2VyO1xuXG5FU0xpbnQiLCJqc29uIjoiIiwibWFya2Rvd24iOiI8Y29kZT5PYmplY3QuY3JlYXRlWzx2YXI-cHJveHk8L3Zhcj5dKGZvbyk8L2NvZGU-IiwiY3NzIjoiQGtleWZyYW1lcyB0ZXN0IHtcblx0MCUge1xuXHRcdG9wYWNpdHk6IDA7XG5cdH1cblxuICAgIGZyb20ge1xuXHRcdG9wYWNpdHk6IDA7XG4gICAgfVxuXG5cdDEwMCUge1xuXHRcdG9wYWNpdHk6IDE7XG5cdH1cbn0iLCJodG1sIjoiPCFET0NUWVBFIGh0bWw-XG48IS0tXG5UeXBlIG9yIHBhc3RlIHNvbWUgSFRNTCBoZXJlIHRvIGxlYXJuIG1vcmUgYWJvdXRcbnRoZSBzdGF0aWMgYW5hbHlzaXMgdGhhdCBFU0xpbnQgY2FuIGRvIGZvciB5b3UuXG5cblRoZSB0YWJzIGFyZTpcblxuLSBBU1QgLSBUaGUgQWJzdHJhY3QgU3ludGF4IFRyZWUgb2YgdGhlIGNvZGUsIHdoaWNoIGNhblxuYmUgdXNlZnVsIHRvIHVuZGVyc3RhbmQgdGhlIHN0cnVjdHVyZSBvZiB0aGUgY29kZS4gWW91XG5jYW4gdmlldyB0aGlzIHN0cnVjdHVyZSBhcyBKU09OIG9yIGluIGEgdHJlZSBmb3JtYXQuXG4tLT5cblxuPGh0bWwgbGFuZz1cImVuXCI-XG4gICAgPGhlYWQ-XG4gICAgICAgIDxtZXRhIGNoYXJzZXQ9XCJVVEYtOFwiPlxuICAgICAgICA8dGl0bGU-SFRNTDwvdGl0bGU-XG4gICAgPC9oZWFkPlxuICAgIDxib2R5PlxuICAgICAgICA8cD5UZXh0PC9wPlxuICAgIDwvYm9keT5cbjwvaHRtbD4ifSwibGFuZ3VhZ2UiOiJtYXJrZG93biIsImpzT3B0aW9ucyI6eyJwYXJzZXIiOiJlc3ByZWUiLCJzb3VyY2VUeXBlIjoibW9kdWxlIiwiaXNWZXJzaW9uIjoibGF0ZXN0IiwiaXNKU1giOnRydWV9LCJqc29uT3B0aW9ucyI6eyJqc29uTW9kZSI6Impzb241IiwiYWxsb3dUcmFpbGluZ0NvbW1hcyI6ZmFsc2V9LCJjc3NPcHRpb25zIjp7ImNzc01vZGUiOiJjc3MiLCJ0b2xlcmFudCI6ZmFsc2V9LCJtYXJrZG93bk9wdGlvbnMiOnsibWFya2Rvd25Nb2RlIjoiZ2ZtIiwibWFya2Rvd25Gcm9udG1hdHRlciI6Im9mZiIsIm1hcmtkb3duTWF0aCI6ZmFsc2V9LCJodG1sT3B0aW9ucyI6eyJ0ZW1wbGF0ZUVuZ2luZVN5bnRheCI6Im5vbmUiLCJmcm9udG1hdHRlciI6Im9mZiJ9LCJ3cmFwIjp0cnVlLCJ2aWV3TW9kZXMiOnsiYXN0VmlldyI6InRyZWUiLCJzY29wZVZpZXciOiJmbGF0IiwicGF0aFZpZXciOiJncmFwaCJ9LCJwYXRoSW5kZXgiOnsiaW5kZXgiOjAsImluZGV4ZXMiOjF9LCJlc3F1ZXJ5U2VsZWN0b3IiOnsic2VsZWN0b3IiOiIifX0sInZlcnNpb24iOjB9).

```md
<code>Object.create[<var>proxy</var>](foo)</code>
```

- GitHub example:

<code>Object.create[<var>proxy</var>](foo)</code>

As you can see above, `proxy` becomes a link in GitHub-flavored Markdown as well.

Allowing `Link` syntax inside HTML elements is the original intention of the CommonMark specification, so it cannot be handled in the same way as inline code syntax in Markdown.

We can revisit this issue if https://github.com/DavidAnson/markdownlint/issues/2178 is accepted on the `markdownlint` side, but for now, I don’t think this is a bug, so I’m closing the issue.

---

Also, regarding the linked PR, mdn/translated-content#28472, the relevant content is inside a [CommonMark HTML block](https://spec.commonmark.org/0.31.2/#html-blocks) that begins with `<table class="standard-table">`, which results in a single HTML node in the AST. Therefore, the rule cannot report an error or provide an autofix for this case.

As you can see in the [playground](https://explorer.eslint.org/#eslint-explorer=v2.eyJzdGF0ZSI6eyJ0b29sIjoiYXN0IiwiY29kZSI6eyJqYXZhc2NyaXB0IjoiY29uc3QgaGkgPSA0MjtcblxuZGVidWdnZXI7XG5cbmRlYnVnZ2VyO1xuXG5FU0xpbnQiLCJqc29uIjoiIiwibWFya2Rvd24iOiI8dGFibGUgY2xhc3M9XCJzdGFuZGFyZC10YWJsZVwiPlxuICA8dGhlYWQ-XG4gICAgPHRyPlxuICAgICAgPHRoPu2VuOuTpOufrCAvIO2KuOueqTwvdGg-XG4gICAgICA8dGg-6rCA66Gc7LGE6riwIOuMgOyDgTwvdGg-XG4gICAgPC90cj5cbiAgPC90aGVhZD5cbiAgPHRib2R5PlxuICAgIDx0cj5cbiAgICAgIDx0ZD5cbiAgICAgICAge3tqc3hyZWYoXCJQcm94eS9Qcm94eS9nZXRQcm90b3R5cGVPZlwiLCBcImhhbmRsZXIuZ2V0UHJvdG90eXBlT2YoKVwiKX19XG4gICAgICA8L3RkPlxuICAgICAgPHRkPlxuICAgICAgICB7e2pzeHJlZihcIk9iamVjdC5nZXRQcm90b3R5cGVPZigpXCIpfX08YnIgLz57e2pzeHJlZihcIlJlZmxlY3QuZ2V0UHJvdG90eXBlT2YoKVwiKX19PGJyIC8-e3tqc3hyZWYoXCJPYmplY3QvcHJvdG9cIiwgXCJfX3Byb3RvX19cIil9fTxiciAvPnt7anN4cmVmKFwiT2JqZWN0LnByb3RvdHlwZS5pc1Byb3RvdHlwZU9mKClcIil9fTxiciAvPnt7anN4cmVmKFwiT3BlcmF0b3JzL2luc3RhbmNlb2ZcIiwgXCJpbnN0YW5jZW9mXCIpfX1cbiAgICAgIDwvdGQ-XG4gICAgPC90cj5cbiAgICA8dHI-XG4gICAgICA8dGQ-XG4gICAgICAgIHt7anN4cmVmKFwiUHJveHkvUHJveHkvc2V0UHJvdG90eXBlT2ZcIiwgXCJoYW5kbGVyLnNldFByb3RvdHlwZU9mKClcIil9fVxuICAgICAgPC90ZD5cbiAgICAgIDx0ZD5cbiAgICAgICAge3tqc3hyZWYoXCJPYmplY3Quc2V0UHJvdG90eXBlT2YoKVwiKX19PGJyIC8-e3tqc3hyZWYoXCJSZWZsZWN0LnNldFByb3RvdHlwZU9mKClcIil9fVxuICAgICAgPC90ZD5cbiAgICA8L3RyPlxuICAgIDx0cj5cbiAgICAgIDx0ZD5cbiAgICAgICAge3tqc3hyZWYoXCJQcm94eS9Qcm94eS9pc0V4dGVuc2libGVcIiwgXCJoYW5kbGVyLmlzRXh0ZW5zaWJsZSgpXCIpfX1cbiAgICAgIDwvdGQ-XG4gICAgICA8dGQ-XG4gICAgICAgIHt7anN4cmVmKFwiT2JqZWN0LmlzRXh0ZW5zaWJsZSgpXCIpfX08YnIgLz57e2pzeHJlZihcIlJlZmxlY3QuaXNFeHRlbnNpYmxlKClcIil9fVxuICAgICAgPC90ZD5cbiAgICA8L3RyPlxuICAgIDx0cj5cbiAgICAgIDx0ZD5cbiAgICAgICAge3tqc3hyZWYoXCJQcm94eS9Qcm94eS9wcmV2ZW50RXh0ZW5zaW9uc1wiLCBcImhhbmRsZXIucHJldmVudEV4dGVuc2lvbnMoKVwiKX19XG4gICAgICA8L3RkPlxuICAgICAgPHRkPlxuICAgICAgICB7e2pzeHJlZihcIk9iamVjdC5wcmV2ZW50RXh0ZW5zaW9ucygpXCIpfX08YnIgLz57e2pzeHJlZihcIlJlZmxlY3QucHJldmVudEV4dGVuc2lvbnMoKVwiKX19XG4gICAgICA8L3RkPlxuICAgIDwvdHI-XG4gICAgPHRyPlxuICAgICAgPHRkPlxuICAgICAgICB7e2pzeHJlZihcIlByb3h5L1Byb3h5L2dldE93blByb3BlcnR5RGVzY3JpcHRvclwiLCBcImhhbmRsZXIuZ2V0T3duUHJvcGVydHlEZXNjcmlwdG9yKClcIil9fVxuICAgICAgPC90ZD5cbiAgICAgIDx0ZD5cbiAgICAgICAge3tqc3hyZWYoXCJPYmplY3QuZ2V0T3duUHJvcGVydHlEZXNjcmlwdG9yKClcIil9fTxiciAvPnt7anN4cmVmKFwiUmVmbGVjdC5nZXRPd25Qcm9wZXJ0eURlc2NyaXB0b3IoKVwiKX19XG4gICAgICA8L3RkPlxuICAgIDwvdHI-XG4gICAgPHRyPlxuICAgICAgPHRkPlxuICAgICAgICB7e2pzeHJlZihcIlByb3h5L1Byb3h5L2RlZmluZVByb3BlcnR5XCIsIFwiaGFuZGxlci5kZWZpbmVQcm9wZXJ0eSgpXCIpfX1cbiAgICAgIDwvdGQ-XG4gICAgICA8dGQ-XG4gICAgICAgIHt7anN4cmVmKFwiT2JqZWN0LmRlZmluZVByb3BlcnR5KClcIil9fTxiciAvPnt7anN4cmVmKFwiUmVmbGVjdC5kZWZpbmVQcm9wZXJ0eSgpXCIpfX1cbiAgICAgIDwvdGQ-XG4gICAgPC90cj5cbiAgICA8dHI-XG4gICAgICA8dGQ-XG4gICAgICAgIHt7anN4cmVmKFwiUHJveHkvUHJveHkvaGFzXCIsIFwiaGFuZGxlci5oYXMoKVwiKX19XG4gICAgICA8L3RkPlxuICAgICAgPHRkPlxuICAgICAgICA8ZGw-XG4gICAgICAgICAgPHN0cm9uZz7sho3shLEg7KGw7ZqMPC9zdHJvbmc-XG4gICAgICAgICAgPGRkPjxjb2RlPmZvbyBpbiBwcm94eTwvY29kZT48L2RkPlxuICAgICAgICAgIDxzdHJvbmc-7IOB7IaNIOyGjeyEsSDsobDtmow8L3N0cm9uZz5cbiAgICAgICAgICA8ZGQ-XG4gICAgICAgICAgICA8Y29kZT5mb28gaW4gT2JqZWN0LmNyZWF0ZSg8dmFyPnByb3h5PC92YXI-KTwvY29kZVxuICAgICAgICAgICAgPjxiciAvPnt7anN4cmVmKFwiUmVmbGVjdC5oYXMoKVwiKX19XG4gICAgICAgICAgPC9kZD5cbiAgICAgICAgPC9kbD5cbiAgICAgIDwvdGQ-XG4gICAgPC90cj5cbiAgICA8dHI-XG4gICAgICA8dGQ-XG4gICAgICAgIHt7anN4cmVmKFwiUHJveHkvUHJveHkvZ2V0XCIsIFwiaGFuZGxlci5nZXQoKVwiKX19XG4gICAgICA8L3RkPlxuICAgICAgPHRkPlxuICAgICAgICA8ZGw-XG4gICAgICAgICAgPHN0cm9uZz7sho3shLEg7KCR6re8PC9zdHJvbmc-XG4gICAgICAgICAgPGRkPlxuICAgICAgICAgICAgPGNvZGU-PHZhcj5wcm94eTwvdmFyPltmb29dPC9jb2RlPjxiciAvPjxjb2RlXG4gICAgICAgICAgICAgID48dmFyPnByb3h5PC92YXI-LmJhcjwvY29kZVxuICAgICAgICAgICAgPlxuICAgICAgICAgIDwvZGQ-XG4gICAgICAgICAgPHN0cm9uZz7sg4Hsho0g7IaN7ISxIOygkeq3vDwvc3Ryb25nPlxuICAgICAgICAgIDxkZD5cbiAgICAgICAgICAgIDxjb2RlPk9iamVjdC5jcmVhdGUoPHZhcj5wcm94eTwvdmFyPikmIzgyMDM7W2Zvb108L2NvZGVcbiAgICAgICAgICAgID48YnIgLz57e2pzeHJlZihcIlJlZmxlY3QuZ2V0KClcIil9fVxuICAgICAgICAgIDwvZGQ-XG4gICAgICAgIDwvZGw-XG4gICAgICA8L3RkPlxuICAgIDwvdHI-XG4gICAgPHRyPlxuICAgICAgPHRkPlxuICAgICAgICB7e2pzeHJlZihcIlByb3h5L1Byb3h5L3NldFwiLCBcImhhbmRsZXIuc2V0KClcIil9fVxuICAgICAgPC90ZD5cbiAgICAgIDx0ZD5cbiAgICAgICAgPGRsPlxuICAgICAgICAgIDxzdHJvbmc-7IaN7ISxIO2VoOuLuTwvc3Ryb25nPlxuICAgICAgICAgIDxkZD5cbiAgICAgICAgICAgIDxjb2RlPjx2YXI-cHJveHk8L3Zhcj5bZm9vXSA9IGJhcjwvY29kZT48YnIgLz48Y29kZVxuICAgICAgICAgICAgICA-PHZhcj5wcm94eTwvdmFyPi5mb28gPSBiYXI8L2NvZGVcbiAgICAgICAgICAgID5cbiAgICAgICAgICA8L2RkPlxuICAgICAgICAgIDxzdHJvbmc-7IOB7IaNIOyGjeyEsSDtlaDri7k8L3N0cm9uZz5cbiAgICAgICAgICA8ZGQ-XG4gICAgICAgICAgICA8Y29kZT5PYmplY3QuY3JlYXRlKDx2YXI-cHJveHk8L3Zhcj4pJiM4MjAzO1tmb29dID0gYmFyPC9jb2RlXG4gICAgICAgICAgICA-PGJyIC8-e3tqc3hyZWYoXCJSZWZsZWN0LnNldCgpXCIpfX1cbiAgICAgICAgICA8L2RkPlxuICAgICAgICA8L2RsPlxuICAgICAgPC90ZD5cbiAgICA8L3RyPlxuICAgIDx0cj5cbiAgICAgIDx0ZD5cbiAgICAgICAge3tqc3hyZWYoXCJQcm94eS9Qcm94eS9kZWxldGVQcm9wZXJ0eVwiLCBcImhhbmRsZXIuZGVsZXRlUHJvcGVydHkoKVwiKX19XG4gICAgICA8L3RkPlxuICAgICAgPHRkPlxuICAgICAgICA8ZGw-XG4gICAgICAgICAgPHN0cm9uZz7sho3shLEg7IKt7KCcPC9zdHJvbmc-XG4gICAgICAgICAgPGRkPlxuICAgICAgICAgICAgPGNvZGU-ZGVsZXRlIDx2YXI-cHJveHk8L3Zhcj5bZm9vXTwvY29kZT48YnIgLz48Y29kZVxuICAgICAgICAgICAgICA-ZGVsZXRlIDx2YXI-cHJveHk8L3Zhcj4uZm9vPC9jb2RlXG4gICAgICAgICAgICA-PGJyIC8-e3tqc3hyZWYoXCJSZWZsZWN0LmRlbGV0ZVByb3BlcnR5KClcIil9fVxuICAgICAgICAgIDwvZGQ-XG4gICAgICAgIDwvZGw-XG4gICAgICA8L3RkPlxuICAgIDwvdHI-XG4gICAgPHRyPlxuICAgICAgPHRkPlxuICAgICAgICB7e2pzeHJlZihcIlByb3h5L1Byb3h5L293bktleXNcIiwgXCJoYW5kbGVyLm93bktleXMoKVwiKX19XG4gICAgICA8L3RkPlxuICAgICAgPHRkPlxuICAgICAgICB7e2pzeHJlZihcIk9iamVjdC5nZXRPd25Qcm9wZXJ0eU5hbWVzKClcIil9fTxiciAvPnt7anN4cmVmKFwiT2JqZWN0LmdldE93blByb3BlcnR5U3ltYm9scygpXCIpfX08YnIgLz57e2pzeHJlZihcIk9iamVjdC5rZXlzKClcIil9fTxiciAvPnt7anN4cmVmKFwiUmVmbGVjdC5vd25LZXlzKClcIil9fVxuICAgICAgPC90ZD5cbiAgICA8L3RyPlxuICAgIDx0cj5cbiAgICAgIDx0ZD5cbiAgICAgICAge3tqc3hyZWYoXCJQcm94eS9Qcm94eS9hcHBseVwiLCBcImhhbmRsZXIuYXBwbHkoKVwiKX19XG4gICAgICA8L3RkPlxuICAgICAgPHRkPlxuICAgICAgICA8Y29kZT5wcm94eSguLmFyZ3MpPC9jb2RlXG4gICAgICAgID48YnIgLz57e2pzeHJlZihcIkZ1bmN0aW9uLnByb3RvdHlwZS5hcHBseSgpXCIpfX0g67CPXG4gICAgICAgIHt7anN4cmVmKFwiRnVuY3Rpb24ucHJvdG90eXBlLmNhbGwoKVwiKX19PGJyIC8-e3tqc3hyZWYoXCJSZWZsZWN0LmFwcGx5KClcIil9fVxuICAgICAgPC90ZD5cbiAgICA8L3RyPlxuICAgIDx0cj5cbiAgICAgIDx0ZD5cbiAgICAgICAge3tqc3hyZWYoXCJQcm94eS9Qcm94eS9jb25zdHJ1Y3RcIiwgXCJoYW5kbGVyLmNvbnN0cnVjdCgpXCIpfX1cbiAgICAgIDwvdGQ-XG4gICAgICA8dGQ-XG4gICAgICAgIDxjb2RlPm5ldyBwcm94eSguLi5hcmdzKTwvY29kZVxuICAgICAgICA-PGJyIC8-e3tqc3hyZWYoXCJSZWZsZWN0LmNvbnN0cnVjdCgpXCIpfX1cbiAgICAgIDwvdGQ-XG4gICAgPC90cj5cbiAgPC90Ym9keT5cbjwvdGFibGU-XG4iLCJjc3MiOiJAa2V5ZnJhbWVzIHRlc3Qge1xuXHQwJSB7XG5cdFx0b3BhY2l0eTogMDtcblx0fVxuXG4gICAgZnJvbSB7XG5cdFx0b3BhY2l0eTogMDtcbiAgICB9XG5cblx0MTAwJSB7XG5cdFx0b3BhY2l0eTogMTtcblx0fVxufSIsImh0bWwiOiI8IURPQ1RZUEUgaHRtbD5cbjwhLS1cblR5cGUgb3IgcGFzdGUgc29tZSBIVE1MIGhlcmUgdG8gbGVhcm4gbW9yZSBhYm91dFxudGhlIHN0YXRpYyBhbmFseXNpcyB0aGF0IEVTTGludCBjYW4gZG8gZm9yIHlvdS5cblxuVGhlIHRhYnMgYXJlOlxuXG4tIEFTVCAtIFRoZSBBYnN0cmFjdCBTeW50YXggVHJlZSBvZiB0aGUgY29kZSwgd2hpY2ggY2FuXG5iZSB1c2VmdWwgdG8gdW5kZXJzdGFuZCB0aGUgc3RydWN0dXJlIG9mIHRoZSBjb2RlLiBZb3VcbmNhbiB2aWV3IHRoaXMgc3RydWN0dXJlIGFzIEpTT04gb3IgaW4gYSB0cmVlIGZvcm1hdC5cbi0tPlxuXG48aHRtbCBsYW5nPVwiZW5cIj5cbiAgICA8aGVhZD5cbiAgICAgICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgICAgIDx0aXRsZT5IVE1MPC90aXRsZT5cbiAgICA8L2hlYWQ-XG4gICAgPGJvZHk-XG4gICAgICAgIDxwPlRleHQ8L3A-XG4gICAgPC9ib2R5PlxuPC9odG1sPiJ9LCJsYW5ndWFnZSI6Im1hcmtkb3duIiwianNPcHRpb25zIjp7InBhcnNlciI6ImVzcHJlZSIsInNvdXJjZVR5cGUiOiJtb2R1bGUiLCJpc1ZlcnNpb24iOiJsYXRlc3QiLCJpc0pTWCI6dHJ1ZX0sImpzb25PcHRpb25zIjp7Impzb25Nb2RlIjoianNvbjUiLCJhbGxvd1RyYWlsaW5nQ29tbWFzIjpmYWxzZX0sImNzc09wdGlvbnMiOnsiY3NzTW9kZSI6ImNzcyIsInRvbGVyYW50IjpmYWxzZX0sIm1hcmtkb3duT3B0aW9ucyI6eyJtYXJrZG93bk1vZGUiOiJnZm0iLCJtYXJrZG93bkZyb250bWF0dGVyIjoib2ZmIiwibWFya2Rvd25NYXRoIjpmYWxzZX0sImh0bWxPcHRpb25zIjp7InRlbXBsYXRlRW5naW5lU3ludGF4Ijoibm9uZSIsImZyb250bWF0dGVyIjoib2ZmIn0sIndyYXAiOnRydWUsInZpZXdNb2RlcyI6eyJhc3RWaWV3IjoidHJlZSIsInNjb3BlVmlldyI6ImZsYXQiLCJwYXRoVmlldyI6ImdyYXBoIn0sInBhdGhJbmRleCI6eyJpbmRleCI6MCwiaW5kZXhlcyI6MX0sImVzcXVlcnlTZWxlY3RvciI6eyJzZWxlY3RvciI6IiJ9fSwidmVyc2lvbiI6MH0), it’s only wrapped in a single HTML node, so it’s unaffected by this rule.
